environment.rb

```diff
@@ -0,0 +1,101 @@
+module Environment
+  class Depot
+    attr_accessor :packs
+
+    def initialize
+      @packs = {
+        simple_transportation_pack: {
+          intelligence: [:cellphone],
+          item: [
```

Comment:
Some code would be better if written using as few lines as possible, I would write this array as:
`item : [:medipack, :chevy_versa],`